src/ui/snipping_tool.py

```diff
@@ -1,33 +1,52 @@
 import tkinter as tk
-from tkinter import Tk, Canvas
+from tkinter import Tk, Canvas, PhotoImage
 from PIL import Image
 from loguru import logger
 from typing import Optional
 import abc
+import os
+import sys
+import time
 
 from src.services.qr_processor import QRCodeProcessor
 from src.services.screenshot_service import ScreenshotCapture
 from src.services.clipboard_service import ClipboardService
 from src.services.notification_service import NotificationService
 
 
+def get_asset_path(relative_path):
```

Comment:
I noticed that the `get_asset_path` function is repeated in both **src/ui/snipping_tool.py** and **main.py**. To prevent code duplication and improve maintainability, would it make sense to move this function to a new utility module in the **utils** folder? 

Let me know your thoughts on this approach!

Reply:
Yes, it's a good idea.

Reply:
Everything else looks good to me. If you'd like to work on this, feel free to let me know. Otherwise, I can go ahead and merge the PR.

Reply:
Done